src/_includes/compass_teaser.html

```diff
@@ -1,6 +1,6 @@
 <section class="{{ include.class }}">
   <div class="compass-teaser">
-    {%- assign anchor_id = {site.translations[site.lang].software-modernisation.page-sections.compass} -%}
+    {%- assign anchor_id = all-service-lines.page-sections.compass -%}
```

Comment:
How does it get the translations from all-service-lines? Do we need a `{t all-service-lines.page-sections.compass} ` for this?

Reply:
Maybe to make it clearer we could not assign `all-service-lines` in the `service_line.liquid` layout and instead do something like `anchor_id = translations.common-service-line-content` ?

Reply:
Personally I prefer something more simple, like 
```suggestion
    {%- assign anchor_id =  site.translations[site.lang].compass-teaser.anchor-id -%}
```

Reply:
This has been actioned now. The Compass Teaser component now pulls in it's anchor independently from the service lines. 

I've also changed it to pull in it's text from a compass-teaser entry within the translation files as it's independent of the service lines at this point too.